File: base64_and_file/src/main/java/com/phone/base64_and_file/presenter/IBase64AndFilePresenter.java
package com.phone.base64_and_file.presenter;

import android.content.Context;

import com.phone.base64_and_file.bean.Base64AndFileBean;

public interface IBase64AndFilePresenter {

    void showCompressedPicture(Context context,
                               Base64AndFileBean base64AndFileBean);

    void showPictureToBase64(Base64AndFileBean base64AndFileBean);

    void showBase64ToPicture(Base64AndFileBean base64AndFileBean);

}

File: library_base64_and_file/src/main/java/com/phone/library_base64_and_file/presenter/IBase64AndFilePresenter.kt
package com.phone.library_base64_and_file.presenter

import android.content.Context
import com.phone.library_base64_and_file.bean.Base64AndFileBean

interface IBase64AndFilePresenter {

    fun showCompressedPicture(
        context: Context,
        base64AndFileBean: Base64AndFileBean
    )

    fun showPictureToBase64(base64AndFileBean: Base64AndFileBean)

    fun showBase64ToPicture(base64AndFileBean: Base64AndFileBean)

}